validator/keymanager/imported/keymanager.go

```diff
@@ -261,15 +265,28 @@ func (dr *Keymanager) DeleteAccounts(ctx context.Context, publicKeys [][]byte) e
 	return nil
 }
 
-// FetchValidatingPublicKeys fetches the list of public keys from the imported account keystores.
+// FetchValidatingPublicKeys fetches the list of active public keys from the imported account keystores.
 func (dr *Keymanager) FetchValidatingPublicKeys(ctx context.Context) ([][48]byte, error) {
 	ctx, span := trace.StartSpan(ctx, "keymanager.FetchValidatingPublicKeys")
 	defer span.End()
 
 	lock.RLock()
 	keys := orderedPublicKeys
-	result := make([][48]byte, len(keys))
-	copy(result, keys)
+	disabledPublicKeys := strings.Split(dr.KeymanagerOpts().DisabledPublicKeys, ",")
```

Comment:
Agreed with the logic, it will become a lot easier if we just use [][]byte instead of `string`